vzzjav/Session_04/dict_lab.py

```diff
@@ -0,0 +1,50 @@
+# Javier Vazquez
+# Grid Printer
+# Jan 29, 2017
+# Description: http://uwpce-pythoncert.github.io/IntroPython2016a/exercises/dict_lab.html
+
+import os, string
+
+def main():
+    cakeDict = {"name": "Chris", "city": "Seattle", "cake": "Chocolate"}
+    print(cakeDict)
+    del cakeDict["cake"]
+    print(cakeDict)
+    cakeDict["fruit"] = "Mango"
+    print(cakeDict)
+    print(cakeDict.keys())
+    print(cakeDict.values())
+    print("cake" in cakeDict.keys())
+    print("Mango" in cakeDict.values())
+    cakeDict = {"name": "Chris", "city": "Seattle", "cake": "Chocolate"}
+    i = 0
+    for key in cakeDict.keys():
+        cakeDict[key] = i
+        i += 1
+    print(cakeDict)
```

Comment:
Can you try to convert this for loop to a dictionary comprehension? As a hint, you can use `enumerate` to act as your counter, so no need for `i=0` and `i += 1`